src/applications/disability-benefits/686c-674/actions/index.js

```diff
@@ -10,12 +11,10 @@ export const VERIFY_VA_FILE_NUMBER_FAILED = 'VERIFY_VA_FILE_NUMBER_FAILED';
 
 // This will eventually take apiRoute and options as it's params.
 // stub out a response for now.
-const getVaFileNumber = () =>
-  new Promise(resolve => {
-    setTimeout(() => {
-      resolve({ data: { attributes: true } });
-    }, 4000);
-  });
+async function getVaFileNumber() {
+  const response = await getData('/profile/va_file_number');
+  return response;
+}
 
 // stub out error response
 // const getVaFileNumber = () =>
```

Comment:
This can probably be removed, now that we're using the real endpoint

Reply:
agreed. Removed.